lib/wechat_pay/plug/refund.ex

```diff
@@ -125,4 +119,17 @@ defmodule WechatPay.Plug.Refund do
     {:error,
      %Error{reason: "Missing the encrypted `req_info` in response data", type: :missing_req_info}}
   end
+
+  # https://erlang.org/doc/apps/crypto/new_api.html#the-new-api
+  if System.otp_release() |> String.to_integer() >= 23 do
```

Comment:
Is checking the OTP version better than checking the specific function exported? Please elaborate.

I personally would prefer a check with `function_exported?`.

Reply:
Updated